app/assets/stylesheets/views/_landing_page/featured.scss

```diff
@@ -2,17 +2,28 @@
 
 .featured {
   display: flex;
-  background: govuk-colour("dark-blue");
 
   @include govuk-media-query($until: desktop) {
     flex-direction: column;
   }
 }
 
 .featured__child--content {
+  width: 66.666%;
+
+  @include govuk-media-query($until: desktop) {
+    width: 100%;
+  }
+}
+
+.featured__child--content-with-image {
   flex: 1 1 33.333%;
-  padding: govuk-spacing(6);
+  padding: 0 govuk-spacing(6) govuk-spacing(6) 0;
   box-sizing: border-box;
+
+  @include govuk-media-query($until: desktop) {
+    padding-bottom: 0;
+  }
 }
 
 .featured__child--image {
```

Comment:
I see that the image scales within the container, and any overflow is clipped. Just wanted to confirm if that's okay or if we want the image to maintain its aspect ratio instead?

Reply:
Good question, I'm not sure - it's looking like it currently works like that on `main` (i.e. gets clipped) atm, I can check with folks to confirm